uwblueprint/planet-read
# Review progress (Backend changes)

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Retrieve-and-display-review-progress-035c3bc67b174c0190a61392ba2d0bef)



## Implementation description
* Create new query to return the number of approved lines 
* Change ProgressBar component to take in name (Review/Translation) as a parameter 
* Added new state numApprovedLines to ReviewPage 


## Steps to test
1. Go to a review page and check to see that there is a Review Progress bar with the correct progress indicated 



## What should reviewers focus on?
* There are not that many changes, make sure that the review progress bar is displayed correctly and nothing is breaking 


## Checklist
- [x] For backend changes, I have run the appropriate linters:  `docker exec -it <backend-container-id> /bin/bash -c "black . && isort --profile black ."` and I have generated new migrations: `flask db migrate -m "<your message>"`
